src/timedata.h

```diff
@@ -73,6 +73,7 @@ class CMedianFilter
 /** Functions to keep track of adjusted P2P time */
 int64_t GetTimeOffset();
 int64_t GetAdjustedTime();
+int64_t GetLocalTime();
```

Comment:
what is this? I'd say please use `GetTime` or (if possible) `GetTime<chrono>`

Reply:
The idea of `GetLocalTime` was to encapsulate `count_seconds(GetTime<std::chrono::seconds>()` to avoid repetition.

I removed GetLocalTime` function, but `GetTime<chrono>` can't be used without `count_seconds` since `GetAdjustedTime` returns `int_64`.